### DISCUSSION TITLE
Multiple active links when having lenses and resources in custom menus

### DISCUSSION THREAD

**Discussion by andypooletrioteca** [laravel/nova-issues#5421] [Ideas & Feature Requests]
- Laravel Version: 10
- Nova Version: 4.22.2
- PHP Version: 8.2

### Description:
My clients struggle with understanding lenses and resources. They prefer having direct access to lenses through the custom menu instead of selecting them from the resource list. They also want access to the complete resource list. However, when they click on a lens link in the custom menu, it marks both the resource and the lens as active, which can be confusing as it makes it seem like they are on multiple pages at once.

### Detailed steps to reproduce the issue on a fresh Nova installation:
create a resource
add a lens to the resource
create a custom menu
add a link to the resource
add a link to the lens
navigate to the lens

**Comment by davidhemphill** [comment]
Converted back to an issue since it's definitely a bug.
